core/src/main/java/ch/cyberduck/core/io/FileBuffer.java

```diff
@@ -35,21 +35,27 @@ public class FileBuffer implements Buffer {
     private static final Logger log = LogManager.getLogger(FileBuffer.class);
 
     private final Local temporary;
+    private final RandomAccessFile file;
 
-    private RandomAccessFile file;
     private Long length = 0L;
 
-    public FileBuffer() {
+    public FileBuffer() throws AccessDeniedException {
         this(TemporaryFileServiceFactory.get().create(new AlphanumericRandomStringService().random()));
     }
 
-    public FileBuffer(final Local temporary) {
+    public FileBuffer(final Local temporary) throws AccessDeniedException {
         this.temporary = temporary;
+        try {
+            this.file = new RandomAccessFile(temporary.getAbsolute(), "rw");
```

Comment:
Most probably not a good idea to create a `RandomAccessFile` in the constructor as in Mountain Duck for every open operation a new `FileBuffer` is created.